lib/widgets.php

```diff
@@ -110,7 +115,7 @@ function gutenberg_legacy_widget_settings( $settings ) {
 		}
 		$available_legacy_widgets[ $widget_id ] = array(
 			'name'             => html_entity_decode( $widget_obj['name'] ),
-			'description'      => null,
+			'description'      => html_entity_decode( wp_widget_description( $widget_id ) ),
```

Comment:
In what way is this change related to the fix?

Reply:
During these changes, I noticed I missed to pass a description when I added support some simple call back widgets.
But you are right it is not directly related to these changes I separated this change to other PR https://github.com/WordPress/gutenberg/pull/14615.